app/helpers/application_helper.rb

```diff
@@ -1,2 +1,9 @@
 module ApplicationHelper
+
+  def display_name(item)
+    case item.class
```

Comment:
Case statements evaluate with `===`, so you can do things like

```ruby
case some_variable
when String
  puts some_variable
when Integer
  some_variable.succ.to_s
#...
end
```

Reply:
Without needing to say `some_variable.class`, that is.

Reply:
Thanks, I think I've only ever written one once before. Not a real excuse, but thanks!! I'll make the change now.